cmd/containerd-shim-runhcs-v1/service.go

```diff
@@ -464,3 +465,22 @@ func (s *service) DiagPid(ctx context.Context, req *shimdiag.PidRequest) (*shimd
 		Pid: int32(os.Getpid()),
 	}, nil
 }
+
+func (s *service) ComputeProcessorInfo(ctx context.Context, req *querycompute.ComputeProcessorInfoRequest) (*querycompute.ComputeProcessorInfoResponse, error) {
```

Comment:
Can you split this between `service.go` and `service_internal.go`, as the other APIs do?